packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -296,13 +296,15 @@ class CreateCommand extends CreateBase {
       } else if (_getSupportedPlatformsInPlugin(projectDir).isEmpty){
         _printNoPluginMessage();
       }
-      _printAddPlatformMessage(relativePluginPath);
+      _printWarningToEnablePlatforms(requestedPlatforms);
```

Comment:
can we do something like
```
if (_disabledPlatformsRequested().length > 0) {
    _printWarningToEnablePlatforms(_disabledPlatformsRequested());
}
```

The current code reads like we always need to print warning to enable platforms, which is incorrect.

Reply:
Will make these changes

Reply:
Have made these changes.